handlers/guildHandlers.js

```diff
@@ -1,34 +1,33 @@
 function createMutedRole(guild) {
-  if (guild.available) {
+  if (guild.available && !guild.roles.cache.has('On Mute')) {
```

Comment:
It seems like roles.cache.has takes a role id and not role name. Source: https://discordjs.guide/popular-topics/permissions.html#roles-as-bot-permissions
I think one option may be to use roles.cache.some to check for the name.